meson.build

```diff
@@ -0,0 +1,51 @@
+project(
+  'buchstabensuppe', 'c',
+  version : '0.0.0',
+  license : 'BSD-3-Clause',
+  default_options: ['c_std=c99'],
+)
+
+cc = meson.get_compiler('c')
+pc = import('pkgconfig')
+
+# TODO: version constraints
+utf8proc = dependency('libutf8proc')
+harfbuzz = dependency('harfbuzz')
+# TODO: no pkg-config upstream, maybe ask for it?
+schrift = cc.find_library('schrift')
+math = cc.find_library('m')
```

Comment:
On some platforms, the math library is built into libc. We suggest you use `required: false` here:
https://mesonbuild.com/howtox.html#add-math-library-lm-portably